src/binary-search-tree.js

```diff
@@ -5,6 +5,70 @@ class BinarySearchTree {
     this.left = null;
     this.right = null;
   }
+
+  insert(value) {
+    const node = new BinarySearchTree(value);
+    let currentNode = this;
+    while (currentNode) {
+      if (node.value < currentNode.value) {
+        if (currentNode.left === null) {
```

Comment:
Start getting in the habit of using the `!` operator when you want to check if something is undefined or null. It is more concise and more flexible than checking if something is strictly just null.